src/screens/Home/Home.tsx

```diff
@@ -54,15 +119,15 @@ function Home(props: HomeProps) {
       <Link to={'/customers'}>
         <HomeMenuItem
           label="Customer Alerts"
-          amount={1}
+          amount={customerDetails ? customerDetails : 0}
           sublabels={[
             { amount: 1, label: 'Customers to Charge' },
             { amount: 2, label: 'Outstanding Payments' },
```

Comment:
Change the amount in these sublabels to show the correct numbers from your calculations